releases/release-1.32/release-notes/release-notes-draft.json

```diff
@@ -4523,7 +4526,7 @@
   },
   "127976": {
     "commit": "c45f3ab813045e3988df0cefa21cb8547c226463",
-    "text": "The name port of the sidecar was also allowed to be used.",
+    "text": "The named port of the sidecar is also allowed to be used.",
     "markdown": "The name port of the sidecar was also allowed to be used. ([#127976](https://github.com/kubernetes/kubernetes/pull/127976), [@chengjoey](https://github.com/chengjoey))",
```

Comment:
Is this right? `text` doesn't match `markdown` (also, the text doesn't make sense to me).

Reply:
@aojea could you take a look? Would something like this work:

`Fixed problem with named ports not being available when specified in sidecar containers` ?

Reply:
> @aojea @chengjoey could you take a look? Would something like this work:
> 
> `Fixed problem with named ports not being available when specified in sidecar containers` ?

+1, thanks @fsmunoz , i think your release note is better, what do you think, @aojea?

Reply:
Since this is time-sensitive (to put it mildly), I'm taking the decision of updating with my suggestion. If it's not quite right, we can improve it afterwards.

Reply:
LGTM